runtime/queries/nix/highlights.scm

```diff
@@ -1,16 +1,22 @@
 (comment) @comment
 
+("assert") @keyword.control.exception
+
+("or") @keyword.operator
+
+("rec") @keyword.control.repeat
```

Comment:
This is a bit of a nitpick but these can be rewritten without the surrounding `()`, like `"or" @keyword.operator`